pkg/virt-operator/kubevirt_test.go

```diff
@@ -339,6 +339,12 @@ var _ = Describe("KubeVirt Operator", func() {
 
 		// Make sure that all unexpected calls to kubeClient will fail
 		kubeClient.Fake.PrependReactor("*", "*", func(action testing.Action) (handled bool, obj runtime.Object, err error) {
+			r := action.GetResource().Resource
```

Comment:
should this be removed?

Reply:
Sure does! leftovers from debug.
Thanks!

Reply:
looks like this still needs to be remvoed

Reply:
Absolutely. Thanks!